File: ai-layer/app/models/dto/project/project_update_dto.py
from pydantic import BaseModel, Field


class ProjectUpdateDto(BaseModel):
    projectId: int = Field(..., description="项目ID")
    name: str | None = Field(None, min_length=1, description="新项目名")
    newUserId: int | None = Field(None, ge=1, description="被转让者ID")

File: backend/src/main/java/com/example/ai_kanban/domain/dto/projectdto/ProjectUpdateDto.java
package com.example.ai_kanban.domain.dto.projectdto;

import com.example.ai_kanban.permission.annotation.ProjectId;
import lombok.Data;

@Data
public class ProjectUpdateDto {
    @ProjectId
    private Long projectId;
    private String name;
    private Long newUserId;
}
